man/man8/zfs-send.8

```diff
@@ -40,6 +40,7 @@
 .Nm zfs
 .Cm send
 .Op Fl DLPRbcehnpsvw
+.Op Fl X Ar dataset Ns Oo , Ns Ar dataset Oc Ns ...
```

Comment:
again,
```
.Op Fl X Ar dataset Ns Oo , Ns Ar dataset Oc Ns …
```
likewise below

Reply:
If you are talking about the ellipses, I can't type those.

Reply:
The issue is that that is a non-ASCII character.

Reply:
Well, we have to have the initial commit at less than 80 characters, and there's a bunch of other restrictions requiring ASCII conformance.

Also:  I don't use a GUI for programming, and my character set is ASCII.  There are multiple instances of '...':  zfs-allow.8, zed.8, zpool-scrub.8, zfs-program.8, cstyle.1, and zfsprops.7.  And now zfs-send.8.